README.md

```diff
@@ -2,8 +2,12 @@
 based on Tensorflow. DELTA classifies large satellite images with neural networks, automatically handling
 tiling large imagery.
 
-DELTA is currently under active development by the
-[NASA Ames Intelligent Robotics Group](https://ti.arc.nasa.gov/tech/asr/groups/intelligent-robotics/).
+![DELTA example flowchart](delta/../docs/delta_flowchart.png)
+
+# TODO: Get okay from Brian that using this crop of image 1044 is okay
```

Comment:
@bcoltin 
This clip of an image is from one of the WV images in our dataset. Is including this little clip alright?

Reply:
Yeah my understanding is that derived products are fine